hudi-common/src/main/java/org/apache/hudi/common/table/view/TableFileSystemView.java

```diff
@@ -107,6 +107,14 @@ interface SliceViewWithLatestSlice {
      */
     Stream<FileSlice> getLatestFileSlices(String partitionPath);
 
+    /**
+     * Get the latest file slices for a given partition including the inflight ones.
+     *
+     * @param partitionPath The partition path of interest
+     * @return Stream of latest {@link FileSlice} in the partition path.
+     */
+    Stream<FileSlice> getLatestFileSlicesIncludingInflight(String partitionPath);
+
```

Comment:
I didn't see impl for RocksDB fsview, do we need that?

Reply:
No we don't need there. This is an uplevel of existing API.